ibm/service/vpc/data_source_ibm_is_backup_policy_plan.go

```diff
@@ -0,0 +1,213 @@
+// Copyright IBM Corp. 2021 All Rights Reserved.
+// Licensed under the Mozilla Public License v2.0
+
+package vpc
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/IBM-Cloud/terraform-provider-ibm/ibm/flex"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/IBM/vpc-go-sdk/vpcv1"
+)
+
+func DataSourceIBMIsBackupPolicyPlan() *schema.Resource {
+	return &schema.Resource{
+		ReadContext: dataSourceIBMIsBackupPolicyPlanRead,
+
+		Schema: map[string]*schema.Schema{
+			"backup_policy_id": &schema.Schema{
```

Comment:
Would we have support for clone_policy here ?
references: https://pages.github.ibm.com/riaas/api-spec/spec_genesis_scoped/#/Backup%20policies/get_backup_policy_plan

Reply:
clone policy is not part of GA feature